src/helpers.ts

```diff
@@ -118,7 +118,7 @@ export async function createAsset(
 		cap: "100000",
 		feeAmount: "0",
 		paymentCollector: await owner.getAddress(),
-		feeToken: config.oceanTokenAddress,
+		feeToken: ddo.stats.price.tokenAddress ? ddo.stats.price.tokenAddress : config.oceanTokenAddress,	
```

Comment:
i would add the ? around this like:
`  ddo?.stats?.price?.tokenAddress` just to play safe
we might not have stats at all
apart from that looks good

Reply:
Yeah, good idea. Thanks for the feedback